python/lsst/dax/apdb/apdbCassandraSchema.py

```diff
@@ -418,12 +418,16 @@ def makeSchema(
         Parameters
         ----------
         drop : `bool`
-            If True then drop tables before creating new ones.
+            If True then drop tables before creating new ones. Note that
+            only tables are dropped and not the whole keyspace.
         part_range : `tuple` [ `int` ] or `None`
             Start and end partition number for time partitions, end is not
             inclusive. Used to create per-partition DiaObject, DiaSource, and
             DiaForcedSource tables. If `None` then per-partition tables are
             not created.
+        replication_factor : `int`, optional
+            Replication factor used when creating new keyspace, if keyspace
+            already exists its replication factor is not changed.
```

Comment:
This seems like an important distinction. Is there some feedback for whether a new keyspace was created?

Reply:
I made DM-43623 to improve that.